conf/modules.config

```diff
@@ -433,4 +433,11 @@ process {
     withName: AVGCOV {
         ext.args    = "-T ./"
     }
+
+    publishDir = [
+        path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+    ]
+
```

Comment:
```suggestion

```

This will force output for everything, as long as we have our process specified publishDir then this one can be removed